src/components/TypesOfRecipeCards/ReviewCard.jsx

```diff
@@ -0,0 +1,84 @@
+import { PushPin } from "@phosphor-icons/react";
+import { Link } from "react-router-dom";
+import { pinWhite, pinSize } from "../../shared/style";
+
+const ReviewCard = ({ recipe, review, pinned }) => {
```

Comment:
So you just copied RecipeCard here. Now the code is duplicated. I would use a shared component.